### DISCUSSION TITLE
Per Item Redirect URL

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2355] [Ideas / Feature Requests]
It would be nice to optionally set a custom redirect URL for each product. This way, where the user will be redirected after the purchase depends on the product he bought, and it's not the same for all products in the app.

Probably not a feature for the Core BTCPay, but maybe something for a plugin / bundle with other similar features?

**Comment by Kukks** [comment]
You could have one standard page where you handle the redirection yourself
based on the invoice id (you can add {{InvoiceId}} to the redirection URL
IIRC)

**Comment by ghost** [comment]
Humble feature request bump.

**Comment by bolatovumar** [comment]
@MaxHillebrand does the suggestion from @Kukks above not work?

**Reply by ghost** [reply]
I might not fully understand how to apply the suggestion. With some concrete example, maybe indeed this solves my problem.

**Reply by bolatovumar** [reply]
@MaxHillebrand I think the suggestion is to add `{{InvoiceId}}` to redirect URL in BTCPay and then on your end use the Greenfield API to look up the invoice with the ID that was passed to the redirect URL. Then you can decide where to redirect the user next based on the info that the API returns for this invoice id.
